pom.xml

```diff
@@ -72,6 +72,7 @@
         <module>kotlin</module>
         <module>bom</module>
         <module>math</module>
+        <module>coverage</module>
```

Comment:
Hum, no other way than that of adding a module? 😄

Reply:
Somehow every other thing I tried, I ended up with empty reports. Anyways it was just a test. I'll fix this flaky test first and prepare the PR properly.